ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -1,249 +1,24 @@
 import React from 'react'
-import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
-import { FormSpy } from 'react-final-form'
 import styled from 'styled-components'
-import { Header, List, Grid } from 'semantic-ui-react'
+import { Header, Grid } from 'semantic-ui-react'
 
-import { getElasticsearchEnabled } from 'redux/selectors'
-import { ButtonLink } from 'shared/components/StyledComponents'
 import { configuredField } from 'shared/components/form/FormHelpers'
-import { Select } from 'shared/components/form/Inputs'
-import Modal from 'shared/components/modal/Modal'
-import VariantSearchFormPanels, {
-  HGMD_PATHOGENICITY_PANEL, ANNOTATION_PANEL, FREQUENCY_PANEL, LOCATION_PANEL, QUALITY_PANEL, IN_SILICO_PANEL,
-  annotationFieldLayout, inSilicoFieldLayout, JsonSelectPropsWithAll,
-} from 'shared/components/panel/search/VariantSearchFormPanels'
-import {
-  HIGH_MODERATE_IMPACT_GROUPS, CODING_OTHER_IMPACT_GROUPS, ANNOTATION_OVERRIDE_GROUPS, ANY_PATHOGENICITY_FILTER,
-  SV_GROUPS, SNP_FREQUENCIES, SNP_QUALITY_FILTER_FIELDS, PATHOGENICITY_FIELDS, PATHOGENICITY_FILTER_OPTIONS,
-  MITO_FREQUENCIES, MITO_QUALITY_FILTER_FIELDS, SV_FREQUENCIES, SV_QUALITY_FILTER_FIELDS,
-} from 'shared/components/panel/search/constants'
-import {
-  ALL_INHERITANCE_FILTER, DATASET_TYPE_SNV_INDEL_CALLS, DATASET_TYPE_SV_CALLS, NO_SV_IN_SILICO_GROUPS, VEP_GROUP_SV_NEW,
-  DATASET_TYPE_MITO_CALLS,
-} from 'shared/utils/constants'
+import VariantSearchFormPanels from './VariantSearchFormPanels'
 import { SavedSearchDropdown } from './SavedSearch'
-import LocusListSelector from './filters/LocusListSelector'
-import CustomInheritanceFilter from './filters/CustomInheritanceFilter'
 import ProjectFamiliesField from './filters/ProjectFamiliesField'
-import {
-  INHERITANCE_FILTER_JSON_OPTIONS,
-  INHERITANCE_FILTER_LOOKUP,
-  INHERITANCE_MODE_LOOKUP,
-  ALL_RECESSIVE_INHERITANCE_FILTERS,
-  NUM_ALT_OPTIONS,
-} from '../constants'
-import { getDatasetTypes, getHasHgmdPermission } from '../selectors'
 
 const SavedSearchColumn = styled(Grid.Column)`
   font-size: 0.75em;
 `
 
-const BaseDetailLink = styled(ButtonLink)`
-  &.ui.button.basic {
-    margin-left: .2em;
-    margin-right: 0;
-    font-weight: initial;
-    font-style: inherit;
-  }
-`
-const DetailLink = props => <BaseDetailLink {...props} />
-
 const SAVED_SEARCH_FIELD = {
   name: 'search',
   component: SavedSearchDropdown,
   format: val => val || {},
 }
 
-const INHERITANCE_PANEL = {
```

Comment:
Now that we no longer need the variant panels to be configurable we can clean up the logic for them considerably. Most of the code removed from this file is moved to `VariantSearchFormPanelConfigs`